code/game/objects/items/weapons/manuals.dm

```diff
@@ -499,6 +499,165 @@
 	title = "Space Law"
 	wiki_page = "Space_Law"
 
+/obj/item/weapon/book/manual/security_antag_guide	//if you wanna edit, just copypaste the dat into https://www.w3schools.com/html/tryit.asp?filename=tryhtml_default
+	name = "Enemies of Nanotrasen: A Quick Overview"
+	desc = "Basic knowledge on how to identify and deal with the various intelligent threats to the station."
+	icon_state = "bookAntagGuide"
+	item_state = "bookAntagGuide"
+	author = "Nanotrasen"
+	title = "Enemies of Nanotrasen: A Quick Overview"
+	book_width = 692
+
+/obj/item/weapon/book/manual/security_antag_guide/New(turf/loc)
+	..()
+	dat = {"<html>
+				<head>
+				<style>
+				h1 {font-size: 32px; margin: 15px 0px 5px;color:#F7D61A;background-color: #990000;}
+				h2 {font-size: 24px; margin: 15px 0px 5px;color:#F7D61A;background-color: #990000;}
+				h3 {font-size: 18px; margin: 15px 0px 5px;}
+				li {margin: 2px 0px 2px 15px;}
+				ul {list-style: none; margin: 5px; padding: 0px;}
+				ol {margin: 5px; padding: 0px 15px;}
+				p  {text-indent: 25;}
+				a:link {color: #F7D61A; background-color: transparent; text-decoration: none;}
+				a:visited {color: #F7D61A; background-color: transparent; text-decoration: none;}
+				</style>
+				</head>
+				<body style="font-size: 18px;color:#F7D61A;background-color:#221111;">
+
+				<h1><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'>Enemies of Nanotrasen: A Quick Overview<img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'></h1>
+                <h2>Foreword</h2>
+				<p>A Mega-Corporation the size of Nanotrasen doesn't carve itself a piece of the intergalactic cake without making itself a few enemies along the way. Identifying agents of those enemy factions and thwarting their plans falls into the job description of our Security forces, but every Nanotrasen employee is encouraged to report any suspicious activities, and should the situation call for it, take up arms against them. These antagonistic forces are dangerous and often use underhanded tactics, this guide aims to even the ground even slightly by giving you some information on how they operate, and how they can be dealt with.
```

Comment:
A mega-corporation as successful as Nanotrasen does not cut itself a slice of the intergalactic cake without attracting the attention of undesirable forces. Identifying and neutralising threats to the station and other company assets is the prime responsibility of Security forces, however every employee is expected to do their part in protecting the station to which they have been assigned. This guide serves as a primer on various dangers you may encounter in the course of your duties.

^ is this splitting hairs? I feel like using the word "enemy" implies there's a war or something, but depending on how propaganda-heavy we imagine NT to be, I could see them trying to discredit the Syndicate and such by just referring to them as "terrorists" or something similar